web/client/components/import/dragZone/LoadingContent.jsx

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2018, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const React = require('react');
+
+const Loader = require('../../misc/Loader');
+const Message = require('../../I18N/Message');
+
+module.exports = () => (<div style={{
```

Comment:
Do we have to define a new component in a new file anytime we need a loader?

Reply:
It is reusing the general Loader. This is the view for the view for the fullscreen drop zone. See the mockup